### DISCUSSION TITLE
AMD driver update problem

### DISCUSSION THREAD

**Discussion by DavidRuales** [bnpr/Malt#487] [Q&A]
Hello, I want to report a problem, Malt was working fine until I updated the AMD drivers
https://www.amd.com/en/support/kb/faq/rsx2-install#faq-Download-Software-and-Drivers-
I don't know how to recover my files, [and]([url]([url](url))) I'm in the middle of production right now, can you please help me
![Esc033](https://user-images.githubusercontent.com/44706863/224581532-57382bd8-1fad-44f3-aa3a-9ab3a3c072f3.jpg)

**Comment by DavidRuales** [comment]
**This appears in the console:**

Traceback (most recent call last):
  File "C:\Users\DELL\AppData\Roaming\Blender Foundation\Blender\3.4\scripts\addons\BlenderMalt\MaltRenderEngine.py", line 220, in update_render_passes
    render_outputs = bridge.render_outputs
AttributeError: 'NoneType' object has no attribute 'render_outputs'
updating file_rigging to //esc018.rigging.blend
updating file_shading to //esc018.shading.blend
updating file_modeling to //esc018.modeling.blend
updating file_geometry to //esc018.geometry.blend
updating file_grooming to //esc018.grooming.blend
Python GPU
Python GPU
Traceback (most recent call last):
  File "C:\Users\DELL\AppData\Roaming\Blender Foundation\Blender\3.4\scripts\addons\BlenderMalt\.MaltPath\Bridge\Server.py", line 43, in log_format_prop
    LOG.info('{} {}: {}'.format(GL_ENUMS[format], GL_ENUMS[prop], GL_ENUMS[read]))
KeyError: 36193
Traceback (most recent call last):
  File "C:\Users\DELL\AppData\Roaming\Blender Foundation\Blender\3.4\scripts\addons\BlenderMalt\.MaltPath\Bridge\Server.py", line 43, in log_format_prop
    LOG.info('{} {}: {}'.format(GL_ENUMS[format], GL_ENUMS[prop], GL_ENUMS[read]))
KeyError: 36193

**Comment by pragma37** [comment]
This is likely the same issue reported in #451.

If you need an immediate solution, I think your safest bet would be to revert to the previous driver version you were using.

Also, for the next time, keep in mind Q&A is for how-to questions, not for bug reports.

**Comment by DavidRuales** [comment]
@pragma37, thank you very much for answering. 
Yes, it is the same error
I will take your recommendation into account
Thank you so much.

**Reply by NipXray** [reply]
@DavidRuales, how about installing the development version? its work for me in latest AMD Driver

https://github.com/bnpr/Malt/releases/tag/Development-latest

**Reply by DavidRuales** [reply]
@NipXray,  thanks for the recommendation, this version works!! , I have about four versions of Blender installed, there are many changes, Blender is evolving too fast and that makes GPUs obsolete after three years, it is more expensive than buying licenses that is worrying
